docs/prepare-a-shared-rp-development-environment.md

```diff
@@ -102,8 +102,8 @@ locations.
    utility:
 
    ```bash
-   go run ./hack/genkey -client firstparty-development
-   mv firstparty-development.* secrets
+   go run ./hack/genkey -client firstparty
```

Comment:
Align the names with what is provisioned in production/Int...

Reply:
please manually update our shared secrets when this merges?